packages/rest/src/body-parsers/body-parser.json.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {RequestBodyParserOptions} from '../types';
+
+import {json} from 'body-parser';
+import {inject} from '@loopback/context';
+import {Request} from '../..';
+import {RestBindings} from '../keys';
+import {is} from 'type-is';
+
+import {RequestBody, BodyParser} from './types';
+import {
+  BodyParserWithCallback,
+  DEFAULT_LIMIT,
+  getParserOptions,
+  parseRequest,
+} from './helper';
```

Comment:
I believe we are trying to keep filenames unique so that we don't depend on file paths to distinguish between them. Can you rename this file to something like `body-parser.helpers.ts` please?